contrib/librdkafka-cmake/config.h.in

```diff
@@ -67,7 +66,8 @@
 #cmakedefine WITH_SASL_OAUTHBEARER 1
 #cmakedefine WITH_SASL_CYRUS 1
 // crc32chw
-#if !defined(__PPC__)
+// actually some arch cpus may have hw crc, feature detection can be added.
```

Comment:
BTW we build code that rely on CRC instructions available on ARM.
You can also enable it here in the same way as in Hash.h.

```
#if defined(__aarch64__) && defined(__ARM_FEATURE_CRC32)
```

Reply:
:+1: 
thanks!